content/20.pharmaceuticals.md

```diff
@@ -370,6 +370,16 @@ Both doses were well tolerated with no serious events related to the antibody co
 Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
 Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].
 
+Sotrovimab is the most recent mAb to receive an EUA.
+It was identified in the memory B cells of a 2003 survivor of SARS and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y]. (look at 24, 26, 32)
```

Comment:
Is `(look at 24, 26, 32)` (and also further down) an internal comment, or meant to be there?

Reply:
HAHA thank you for catching this!! It was my note to myself that I never cleaned up! I'll do that this morning 🤦‍♀️